cdap-app-fabric/src/main/java/io/cdap/cdap/internal/provision/task/ProvisionTask.java

```diff
@@ -114,9 +114,9 @@ protected void handleStateSaveFailure(ProvisioningTaskInfo taskInfo, Exception e
   private ProvisioningSubtask createClusterCreateSubtask() {
     return new ClusterCreateSubtask(provisioner, provisionerContext, cluster -> {
       if (!Objects.equals(provisionerContext.getCDAPVersionInfo(), provisionerContext.getAppCDAPVersionInfo())) {
-        USERLOG.info("Starting a pipeline created with a previous CDAP version. " +
+        USERLOG.info("Starting a pipeline created with a previous version. " +
```

Comment:
Any particular reason we are removing references to CDAP?

Reply:
It will be visible in both CDAP and CDF